This patch removes the legacy oslo namespace deprecation check from the
oslo.i18n library. The deprecated() function is left to satisfy the
unit tests. It is not suitable for the upstream and should be removed
once all of the relevant consumers have switched over to the new
namespace.

--- oslo.i18n-1.3.1/oslo/i18n/__init__.py.~1~	2015-01-27 05:34:49.000000000 -0800
+++ oslo.i18n-1.3.1/oslo/i18n/__init__.py	2015-03-02 22:45:07.303224309 -0800
@@ -26,6 +26,3 @@ def deprecated():
         DeprecationWarning,
         stacklevel=3,
     )
-
-
-deprecated()
